jupyter/scripts/incremental_learning/job_submitter.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
+# coding: utf-8
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License
+# 2.0 and the following additional limitation. Functionality enabled by the
+# files subject to the Elastic License 2.0 may only be used in production when
+# invoked by an Elasticsearch process with a license key installed that permits
+# use of machine learning features. You may not use this file except in
+# compliance with the Elastic License 2.0 and the foregoing additional
+# limitation.
+
+import json
+import os
+import subprocess
+from pathlib import Path
+
+from jinja2 import Environment, FileSystemLoader
+
+def init_task_spooler():
+    cmd = ['tsp', '-S', '`nproc`']
+    process = subprocess.Popen(cmd,
+                                stdout=subprocess.PIPE,
+                                stderr=subprocess.PIPE)
+    stdout, stderr = process.communicate()
```

Comment:
How about printing these particularly if return code != 0. Also in `submit_to_task_spooler`.